uninstall.php

```diff
@@ -1,44 +1,120 @@
 <?php
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly.
-}
+/**
+ * WooCommerce Stripe Gateway Uninstall
+ *
+ * @version  x.x.x
```

Comment:
Not sure if we want to define this or if it will bumped once the release happens.